### ISSUE TITLE
Using CRLF breaks breaks sequelize-auto on mac

### ISSUE THREAD

**Issue by sec0ndhand** [sequelize/sequelize-auto#487]
MacOS 10.15.7
node version: v15.3.0
"sequelize": "^6.4.0",
"sequelize-auto": "^0.7.7",

```
env: node\r: No such file or directory
npm ERR! code 127
npm ERR! path /Users/christopherjensen/Development/collective-graphql
npm ERR! command failed
npm ERR! command sh -c sequelize-auto -o ./models -d my_database_name -h localhost -u example -p 5432 -x example -e postgres
```

I fixed this by re-saving the 

./node_modules/sequelize-auto/bin/sequelize-auto

with just LF, and then this worked again.  I don't know if this breaks windows CLI though, but it fixes mac CLI.

**Comment by sbouchard1** [comment]
Stil having the same problem in version 0.7.8.

**Comment by papb** [comment]
> I don't know if this breaks windows CLI though

It definitely does not. Having LF endings is the correct solution. I will see what I can do.

**Comment by steveschmitt** [comment]
I think git converts the line endings to CRLF when I checkout on Windows.  But I could convert the `bin/sequelize-auto` file to LF as part of the npm publishing process.

**Comment by steveschmitt** [comment]
Fixed in 0.7.9
